client/js/jsonedit.js

```diff
@@ -1237,38 +1236,112 @@ function jeColorize() {
   $('#jeTextHighlight').innerHTML = out.join('\n');
 }
 
+/* Displaying and controlling tree subpane of edit area */
+
+const editPanel = document.getElementById("jeEditArea");
+
+let mouse_pos;
+
+function resize(e){
+  function pad(w) {
+    return parseInt(window.getComputedStyle(document.getElementById("jeEditArea")).getPropertyValue(w).slice(0,-2))
+  };
```

Comment:
This seems unnecessary to me. Maybe using  `box-sizing: border-box` somewhere will help? That makes the `width` and `height` you use in CSS _include_ the padding, border and margin (I think).

Reply:
Greatly simplified.